FCMExternalProberTarget/app/src/main/java/com/google/fcmexternalprobertarget/FCMReceiveService.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Google LLC
+ *
```

Comment:
The license header is good, but from the documentation I think it needs to be a header in pretty much every file that supports comments.

"BEST PRACTICE: Add license headers to any files that can be added to (i.e. anything that takes the format of a source file and supports file comments). This would include things like YAML files."

https://opensource.google/docs/releasing/preparing/#license-headers